### PR TITLE
Doc why FinalizeAccount uses order objects

### PR DESCRIPTION
**Pull request by sauloperez**
Read the top-level class documentation for the details.

This will be very helpful for future devs who read the class. It won't come at a suprise what the class does.

EDIT: more context in [community thread](https://community.openfoodnetwork.org/t/finalize-account-invoices/990)

### CODE REVIEW

File: app/jobs/finalize_account_invoices.rb [hunk 1]
```diff
@@ -1,3 +1,18 @@
...
+# The "orders" used by this class are not real orders, they are basically
+# "invoices" that enterprise owners need to pay for use of an open food network
+# instance. The amount that the enterprise owner is charged is configurable by
+# the instance, and can be based on a combination of a percentage of their
+# turnover, fixed fees, caps and floors and trial periods. This "orders" # hold
+# the billing information for a particular enterprise owner for a given month.
+#
+# It was thought the only reason they have a shipping method at all is because
+# there is a validation on Spree::Order that requires it but it turns out that
+# it is not.
```
Comment (oeoeaio) [inline app/jobs/finalize_account_invoices.rb]: Oh, so there isn't a validation that requires a shipping method? Can you created a completed order without a shipping_method?

Comment (sauloperez) [inline app/jobs/finalize_account_invoices.rb]: I'll give it a try

**Comment by sauloperez** [comment]
clarified @oeoeaio

**Comment by sauloperez** [comment]
If you guys agree on the documentation we can merge this.

**Comment by oeoeaio** [comment]
Thanks for clarifying @sauloperez. Ready to merge I think.

**Pull request merged**